refresh-period/README.md

```diff
@@ -0,0 +1,112 @@
+# Temporal Refresh Period Quickstart
+
```

Comment:
It would be great to add a few sentences here explaining what the refresh period is and what it can be useful for.

Reply:
It can basically copy what we have in docs if it makes sense